content/_data/changelogs/weekly.yml

```diff
@@ -5201,27 +5201,58 @@
       - issue: 53462
       - url: https://bugzilla.mozilla.org/show_bug.cgi?id=1370630
         title: Firefox issue 1370630
+    authors:
+      - thomasgl-orange
     message: |-
       Revert changes in forms submission in Jenkins classic UI with Firefox have caused a regression on forms with "file" inputs.
       These were made to anticipate a bugfix in Firefox which has been backed out since.
       (regression in 2.173)
   - type: rfe
     pull: 3894
+    authors:
+      - daniel-beck
     message: |-
       Add telemetry on the use of the 'auto refresh' feature.
   - type: rfe
     pull: 4300
+    authors:
+      - anafke
     message: |-
       Add <code>java.util.concurrent.ConcurrentLinkedDeque</code> to the JEP-200 deserialization whitelist.
   - type: rfe
     pull: 4259
+    authors:
+      - jsoref
     message: |-
       Developer: Introduce <code>Run#getBuildsOverThreshold()</code> method for getting runs above the desired execution result.
-  # pull: 4155 (PR title: Revert "Disable Windows builds and ATH to mitigate the ongoing INFRA issue")
-  # pull: 4253 (PR title: Replace ACL.impersonate with ACL.as)
-  # pull: 4292 (PR title: Update at-since in Javadoc, make script work on Mac OS X)
-  # pull: 4303 too minor (PR title: Use actual logger to record Jenkins home and where it was found)
-  # pull: 4313 (PR title: [JENKINS-58625] Cancel older builds of a given branch)
+
+  - type: minor rfe
+    pull: 4313
+    issue: 58625
+    authors:
+      - jglick
+    message: |-
+      Jenkinsfile: Cancel older builds of a given branch.
+  - type: minor bug
```

Comment:
It covers entries which are not shipped to users, and it may cause confusion (docs, Pipeline, co). Also some changes are not really minor but just do not worth being in the user-facing changelog. What if we just add a `skipped` type or additional entry?

Reply:
@oleg-nenashev how would "skipped" be different from "minor"? Or do you just mean to rename it?

Reply:
"Minor" is more semantic, "skipped" is presentational. The effect is the same. Think of it like CSS:

`li.minor { display: none; }`

(Although there's a point here about changes that go into the jenkins.war and those that don't, a distinction I made explicit in my email – there are minor changes that end up in jenkins.war (e.g. minor contributions to translations), as well as changes that don't. I decided to count the latter as contributions towards a release as well, since Jenkinsfile fixes etc. make other contributions easier, better, or even possible at all.)

Reply:
Ideally I would like to split semantics and presentation so that we can manage visualization type easily without changing the Jenkins core labeling policy. E.g:

```yaml
  - type: rfe
    skipped: true
    pull: 4313
    issue: 58625
  - type: localization
    skipped: true
    pull: 4313
    issue: 58625
  - type: internal
    skipped: true
    pull: 4313
    issue: 58625
```

Alternate option would be to have a `LoggingLevel` equivalent:

``` yaml
  - type: rfe
    level: major
    pull: 4313
    issue: 58625
  - type: bug
    label: normal
    pull: 4313
    issue: 58625
  - type: rfe
    level: minor
    pull: 4313
    issue: 58625
```

It would allow to manage the granularity of changelogs and, for example, display minor changes in the dropdown

Reply:
This is basically the same as the last proposal, except we're collapsing both fields into one and just don't explicitly specify "normal" (assumed if unspecified).

> manage visualization type easily without changing the Jenkins core labeling policy

Does this require more than just `if(labels.has? 'skip-changelog') type = 'minor' end` (or its syntactically valid equivalent)?